### ISSUE TITLE
Status of builds not being sent to GitHub for commits and pull requests

### ISSUE THREAD

**Issue by rajsite** [travis-ci/travis-ci#9440]
For the following repo: ni/VireoSDK

You can see in the commits on master that there are no status checks for commits that are being made after March 28:
https://github.com/ni/VireoSDK/commits/master

The builds are being triggered, for example the following commit:
https://github.com/ni/VireoSDK/commit/7b90437a1343586d2a100fe3386afb236cf58aa2

Triggered this build:
https://travis-ci.org/ni/VireoSDK/builds/361872563

But GitHub is not showing any status coming back (see empty statuses array and total_count of 0):
https://api.github.com/repos/ni/VireoSDK/commits/7b90437a1343586d2a100fe3386afb236cf58aa2/status

I tried to disable and reenable the TravisCI integration on https://travis-ci.org/profile/ni
But the toggle does not actually toggle. I have admin rights to the GitHub repo but I cannot disable and reenable the integration:

![togglebroken](https://user-images.githubusercontent.com/1588923/38281097-8b636094-376e-11e8-86a9-f1643cf00bdb.gif)

My current settings are as follows:
![vireosettings](https://user-images.githubusercontent.com/1588923/38281153-dc2aaa0a-376e-11e8-9ab6-7c14f6464dee.PNG)

Any idea how to get the statuses re-enabled from Travis for commits and pull requests?

**Comment by BanzaiMan** [comment]
Quite peculiar. It appears that, when we try to manage the repo's notifications, GitHub is throwing a 404 somehow. Has anything changed recently in this repo?

**Comment by zarlo** [comment]
we are getting the same with borevitzlab/go-eyepi when we try to deploy to releases

**Comment by BanzaiMan** [comment]
For posterity: @zarlo's 404 is unrelated to this issue.

**Comment by rajsite** [comment]
I'm not aware of any local repo setting changes. This is what the integration tab shows for Travis

![integration2](https://user-images.githubusercontent.com/1588923/38315994-50976376-37ef-11e8-9411-767c840f4f70.PNG)

I clicked the Test Service button but not sure what to expect.

**Comment by rajsite** [comment]
Are there any organization level things that need to be verified? I can contact the organization owners if there is something to check in there.

**Comment by BanzaiMan** [comment]
@rajsite Could you try deleting the hook on GitHub, and resyncing the data at https://travis-ci.org/profile/?

If this doesn't work, we'll get in touch with GitHub for support.

**Comment by rajsite** [comment]
Figured it out after talking to the organization owners.

So the organization previously had a policy that did not require approval for GitHub OAuth integrations. Last week they changed the setting so that OAuth integrations require approval.

When turning on the setting the organization owners didn't see a list of existing integrations in organization settings, so they were unaware of what would be turned off. From the organization user settings > applications > Authorized OAuth Apps tab they didn't see anything (because the organization user account hasn't tried any integrations).

From my user settings > applications > Authorized OAuth Apps tab I see TravisCI (because as a developer I had integrated Travis CI initially). When I go into Travis CI > Organization access then I now had the option to request access for TravisCI.

When my request was approved then TravisCI was enabled for the entire organization (turns out we had many repos effected).

It seems like there is still something that needs to happen to make this a better experience,
Travis just getting 404'd by GitHub and not being able to give Travis CI users feedback was confusing. GitHub no longer showing statuses without any indication as to why was confusing. And Org owners not knowing what effect changing a security setting was going to have was confusing.
